src/pages/ProductList/Component/Dropdown/DropdownItem.scss

```
@@ -0,0 +1,49 @@
+.option-box {
+  margin: 15px 0px 0px 15px;
+  .option-container {
+    box-sizing: border-box;
+    display: flex;
+    flex-flow: row wrap;
+    width: 100%;
+
+    .option-item-wrapper {
+      box-sizing: border-box;
+      margin: 0;
+      flex-direction: row;
+      flex-basis: 50%;
+      flex-grow: 0;
+      max-width: 30%;
+      .option-item {
+        display: flex;
+        margin-bottom: 22px;
+        box-sizing: border-box;
+        .option-check-box {
+          display: block;
+          .option-check-button {
```

새로운 선택자 간에는 한 줄씩 띄워서 가독성 높여주세요! 리뷰 남기지 않은 다른 부분도 수정해주세요!
```suggestion
.option-box {
  margin: 15px 0px 0px 15px;
  
  .option-container {
    box-sizing: border-box;
    display: flex;
    flex-flow: row wrap;
    width: 100%;

    .option-item-wrapper {
      box-sizing: border-box;
      margin: 0;
      flex-direction: row;
      flex-basis: 50%;
      flex-grow: 0;
      max-width: 30%;
      
      .option-item {
        display: flex;
        margin-bottom: 22px;
        box-sizing: border-box;
        
        .option-check-box {
          display: block;
          
          .option-check-button {
```

---

수정 하였습니다.